### ISSUE TITLE
Ghidra cannot disassemble cfinv instruction defined in Armv8.4

### ISSUE THREAD

**Issue by kohnakagawa** [NationalSecurityAgency/ghidra#2710]
**Describe the bug**

Ghidra cannot disassemble cfinv instruction defined in Arm v8.4 specification.

**To Reproduce**
1. Download the binary file attached to this issue.
2. Open this file with Ghidra 9.2
3. The disassembly result must be as follows (I tested it using lldb debugger's disassemble command).

<img width="386" alt="スクリーンショット 2021-01-31 15 46 27" src="https://user-images.githubusercontent.com/5129526/106376828-0c955400-63dc-11eb-8a20-95b344f52cc0.png">

However Ghidra cannot disassemble cfinv instruction.

![スクリーンショット 2021-01-31 15 47 42](https://user-images.githubusercontent.com/5129526/106376829-0c955400-63dc-11eb-86bd-e2338c01dfa0.png)

**Expected behavior**
Ghidra should disassemble cfinv instruction.

**Screenshots**
I show some screenshots at "To Reproduce" section.

**Attachments**
I attached binary file to reproduce this problem.

[a.out.zip](https://github.com/NationalSecurityAgency/ghidra/files/5899301/a.out.zip)

**Environment (please complete the following information):**
 - macOS 11.1
 - OpenJDK 11.0.9.1
 - Ghidra 9.2.1

**Additional context**
cfinv is an Armv8.4 instruction. Other instructions belonging to Armv8.4 are defined, such as RMIF, so I suspect that the developers simply forgot to define it.

**Comment by mumbel** [comment]
see if this helps.  Edit your `Ghidra/Processors/AARCH64/data/languages/AARCH64base.sinc` and insert this just after line  4218 (directly after comment before `msr`) and restart ghidra

```
:cfinv                                                                                               
is b_2431=0xd5 & b_2223=0 & b_1621=0 & CRn=0x4 & CRm=0 & b_0507=0 & Rt=0x1f                          
{                                                                                                    
        nzcv = nzcv ^ 2;                                                                             
} 
```

**Comment by kohnakagawa** [comment]
@mumbel Thank you! I have already implemented as follows.

```
:cfinv
is b_0031=0xd500401f
{
    CY = !CY;
}
```

**Comment by mumbel** [comment]
@kohnakagawa  I'm not familiar enough w/ aarch64 sleigh code and I didn't look too closely on `nzcv` vs `CY`, but it looks like the `nzcv` approach will be the pstate bits also, but now not seeing how they join up in sleigh.  `CY` is probably a much better choice here

**Comment by emteere** [comment]
Currently the flag registers are modeled as separate 1 byte registers, so it should be the CY flag.
Some of the bits in the manual are also (0) which is reserved/should be zero.
The execution is Unpredictable, but could execute as if the bits were 0.
In other instructions, they are not locked down to 1 or 0, unless there is another conflicting instruction that should match.
I defined it as this to match the xaflag and axflag instructions.

:cfinv                                                                                               
is b_1231=0b11010101000000000100 & b_0007=0b00011111
{                                                                                                                           
        CY = !CY;                                                                    
}

There is an auto AARCH64 scraper that marks the docs up that needs to be re-run to auto-comment this.  There are also some "local" keywords missing that should be inserted automatically in the NEON, which causes an error in the editor.

**Comment by ghidra1** [comment]
Fixed for Ghidra 9.2.3
